content/en/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md

```diff
@@ -69,11 +78,90 @@ The script above only detects the most common uses.
 ### Detecting Docker dependency from node agents
 
 In case your cluster nodes are customized and install additional security and
-telemetry agents on the node, make sure to check with the vendor of the agent whether it has dependency on Docker.
+telemetry agents on the node, make sure to check with the vendor of the agent
+whether it has dependency on Docker.
 
 ### Telemetry and security agent vendors
 
+This section is intended to collect information about various telemetry and
+security agents that may have a dependency on container runtimes. The Support
+matrix section outlines the supported runtimes and the Migration from dockershim
+section is designed to help users transition from dockershim to other container
+runtimes.
```

Comment:
There's no support matrix on the page. I think we can safely remove everything after the first sentence.
```suggestion
security agents that may have a dependency on container runtimes.
```